src/test/regress/sql/multi_extension.sql

```diff
@@ -36,6 +36,37 @@ $$;
 $definition$ create_function_test_maintenance_worker
 \gset
 
+CREATE TABLE prev_objects(description text);
+CREATE TABLE extension_diff(previous_object text COLLATE "C",
+                            current_object text COLLATE "C");
+
+CREATE FUNCTION print_extension_changes()
+RETURNS TABLE(previous_object text, current_object text)
+AS $func$
+BEGIN
+	TRUNCATE TABLE extension_diff;
+
+	CREATE TABLE current_objects AS
+	SELECT pg_catalog.pg_describe_object(classid, objid, 0) AS description
```

Comment:
Below are some ideas to improve this diff, by displaying some more info. All not critical for merging this PR, but if there's an easy way to do so it would be nice:
1. The attached `COMMENT`, this one seems like it would be pretty easy to add using `obj_description(...)`.
2. Show result types of functions, possible to get using `pg_get_function_result`.
3. Show full defs of objects by using: `pg_get_functiondef`, `pg_get_triggerdef`, `pg_get_indexdef`, `pg_get_viewdef`, `pg_get_ruledef`
4. Show table columns.